### PR TITLE
Update @ckeditor/ckeditor5-core to v33

### PR DESCRIPTION
**Pull request by fedemp**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-core/src/index.js#L30
https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-core/src/command.js#L57
https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-core/src/command.js#L98
https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-core/src/command.js#L113
https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-core/src/editor/editorconfig.jsdoc#L168
https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-core/src/editor/editorconfig.jsdoc#L314
https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-core/src/editor/editorui.js#L275
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/ckeditor__ckeditor5-core/src/command.d.ts [hunk 1]
```diff
@@ -4,15 +4,161 @@ import Editor from './editor/editor';
...
+     *    command.forceDisabled( 'MyFeature' );
+     *    command.clearForceDisabled( 'MyFeature' );
+     *    command.isEnabled; // -> true
+     *
+     * **Note:** some commands or algorithms may have more complex logic when it comes to enabling or disabling certain commands,
+     * so the command might be still disabled after {@link #clearForceDisabled} was used.
+     */
     forceDisabled(id: string): void;
+
+	/**
```
Comment (DangerBotOSS) [inline types/ckeditor__ckeditor5-core/src/command.d.ts:138]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/ckeditor__ckeditor5-core/src/editor/editorui.d.ts [hunk 2]
```diff
@@ -2,21 +2,108 @@ import ComponentFactory from '@ckeditor/ckeditor5-ui/src/componentfactory';
...
+     * Stores viewport offsets from every direction.
+     *
+     * Viewport offset can be used to constrain balloons or other UI elements into an element smaller than the viewport.
+     * This can be useful if there are any other absolutely positioned elements that may interfere with editor UI.
+     *
+     * Example `editor.ui.viewportOffset` returns:
+     *
+     * ```js
+     * {
+     * 	top: 50,
```
Comment (DangerBotOSS) [inline types/ckeditor__ckeditor5-core/src/editor/editorui.d.ts:45]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/ckeditor__ckeditor5-list/src/checktodolistcommand.d.ts [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
+ */
 export default class CheckTodoListCommand extends Command {
-    value: Element[];
+    /**
+     * A list of to-do list items selected by the {@link module:engine/model/selection~Selection}.
+     */
+    get value(): Element[];
+    protected set value(value: Element[]);
+
+	/**
```
Comment (DangerBotOSS) [inline types/ckeditor__ckeditor5-list/src/checktodolistcommand.d.ts:17]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Review by gabritto — CHANGES_REQUESTED** [review]
Looks good, but I noticed the versions of the packages besides the core one weren't updated to 33. Shouldn't they be?

**Comment by fedemp** [comment]
> Looks good, but I noticed the versions of the packages besides the core one weren't updated to 33. Shouldn't they be?

There are some packages that were updated when v33 was released, and that I haven't got the time to update.

If you mean that every packages should be updated to v33, that's not needed. When you install the type definition, it will install whatever is the latest version, and there may be no changes between currently published definitions and upstream.

**Review by gabritto — APPROVED** [review]

**Comment by fedemp** [comment]
Ready to merge

**Pull request merged**